Ukraine holidays

kasya opened the pull request:
Update Ukraine holidays.
    
      - Add Day of Ukrainian Statehood holiday.
      - Edit Victory Day name.
      - Fix formatting.
      - Add test_2022, to check new holiday (Day of Ukrainian Statehood).

kasya committed:
Update Ukraine holidays.

  - Add Day of Ukrainian Statehood holiday.
  - Edit Victory Day name.
  - Fix formatting.

kasya committed:
Update tests for Ukraine holidays.

  - Add test_2022, to check new holiday (Day of Ukrainian Statehood)

arkid15r commented on holidays/countries/ukraine.py:
```diff
@@ -40,15 +40,15 @@ def _populate(self, year):
+            self[date(year, JAN, 7)] = "Різдво Христове (православне)"
 
         # Women's Day
         if year > 1965:
```
Use >= for consistency

arkid15r commented on holidays/countries/ukraine.py:
```diff
@@ -40,15 +40,15 @@ def _populate(self, year):
         # Easter
         if year >= 1991:
-            self[easter(year, method=EASTER_ORTHODOX)] = "Пасха" " (Великдень)"
+            self[easter(year, method=EASTER_ORTHODOX)] = "Пасха (Великдень)"
```
Великдень (Пасха)

arkid15r commented on holidays/countries/ukraine.py:
```diff
@@ -67,16 +67,22 @@ def _populate(self, year):
 
         # Victory Day
         name = "День перемоги"
-        if year >= 1965:
```
I'd leave it as it was

arkid15r commented on test/countries/test_ukraine.py:
```diff
@@ -34,6 +34,21 @@ def test_2018(self):
+    def test_2022(self):
+        self.assertIn(date(2022, 1, 1), self.holidays)
+        self.assertIn(date(2022, 1, 7), self.holidays)
+        self.assertIn(date(2022, 12, 25), self.holidays)
```
Move to the end

arkid15r reviewed (commented):
`
if year < 1918:
            return
`
seems to be irrelevant

kasya committed:
Resolve PR issues.

  - Switch to <= and >= for consistency.
  - Rename Orthodox Easter.
  - Add assertion for Women's Day.

kasya committed:
Fix pull request issues.

arkid15r reviewed (approved):
LGTM
